README.md

```diff
@@ -1,26 +1,197 @@
 # Rails Layouts And Templates
 
+This lesson will show you how to use layouts to achieve a common look and feel between different views in your app.
+
 ## Objectives
 
+After this lesson, you'll be able to...
+
 1. Identify the default application layout.
 2. Yield to view templates from a layout.
 3. Specify a custom layout in ActionController on a controller level using the `layout` macro and on the action level using the `render :layout => "custom"` option.
 4. Use :layout => false to shut off the layout and only render the view
 
-## Outline
+## Life Without Layouts
+
+Imagine that you're tasked to build an online store app with Rails.
+
+You would probably have a few different views in this app, for example:
+
+1. A list of products
+2. A detail view that shows more info for a selected product
+3. A shopping cart
+
+Across all these views you would want a consistent look, which is called a layout. This layout perhaps contains something like a logo, navigation links, a search bar, and a footer at the bottom that contains some info about the shop.
+
+Without layouts you would need to copy and paste your logo, navigation, search, and footer code into each action's template that you want in your application.
+
+And when you want to make a change to one of the common components, for example the navigation, you would have to go and make that change in all the template files that you created in your application. This will obviously slow you down a lot.
+
```

Comment:
Nice elucidation of the problems that would occur if you didn't use layouts. This is great, but one strategy you can use if you'd like is to not mention the word layout at all. You can talk about the problems as you do here. And really show them the pain; i.e. Show the full HTML of a `show.html.erb` (with all the JS and `head` stuff) then show creating the HTML for the `index.html.erb` page. When you move from show to index, mention "ok so to get started we first copy paste all of the boiler plate code form show". Then mention "Wait a second, copy pasting isn't something we should be doing. That sounds like a task for computers to do. Layouts is the way we get around copy pasting the elements that exist in every page"